app/store/chatList.go

```diff
@@ -331,12 +332,35 @@ func (s *Sessions) GetByUUID(UUID string) (*Session, error) {
 	}
 	return nil, fmt.Errorf("Session with uuid %s not found", UUID)
 }
+func HexToUUID(id string) string {
```

Comment:
Don't you have that function in several places now? Can this be put in an utils file somewhere?